### PR TITLE
Added created date in user entity

### PR DESCRIPTION
**Pull request by Cheshiriks**
- Added created date in user entity
- added registration date on Index view

### CODE REVIEW

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt [hunk 1]
```diff
@@ -81,6 +97,9 @@ val cardUser: FC<IndexViewProps> = FC { props ->
             div {
                 className = ClassName("col-9")
                 props.userInfo?.let {
+                    p {
+                        +"You are with us since ${it.createDate} !"
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt:101]: Need something less aggressive:

Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt:101]: Registered since: DATE

**Review by akuleshov7 — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt [hunk 2]
```diff
@@ -81,6 +97,9 @@ val cardUser: FC<IndexViewProps> = FC { props ->
             div {
                 className = ClassName("col-9")
                 props.userInfo?.let {
+                    p {
+                        +"Registered since ${it.createDate} !"
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt:101]: also I am not sure that create date format is right: need to show year month and date, without time

Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt]: Fixed

File: db/v-2/tables/user.xml [hunk 3]
```diff
@@ -101,5 +101,20 @@
...
+                <constraints nullable="true"/>
+            </column>
+            <column name="update_date" type="DATETIME(3)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+
+        <sql>
+            UPDATE user SET create_date = '2022-01-01 00:00:00';
+            UPDATE user SET create_date = '2022-01-01 00:00:00';
```
Comment (nulls) [inline db/v-2/tables/user.xml:116]: twice `create_date`

**Pull request merged**